frame/election-provider-multi-phase/src/signed.rs

```diff
@@ -52,7 +52,7 @@ pub struct SignedSubmission<AccountId, Balance: HasCompact, Solution> {
 	pub deposit: Balance,
```

Comment:
Also a `#![deny(missing_docs)]` here maybe.

Reply:
Just to be clear, can you explain what leaving this out may lead to in the future?

Reply:
People will just modify the pallet and not add docs. Then you have to perpetually open MRs to fix it up.  
Just adding this attribute will make the MR unmergable without docs on everything.